0.data_analysis/plate_correlation_analyses/nbconverted/well_agg_plate_genotype_correlation_analysis.py

```diff
@@ -167,7 +185,9 @@
 
 
 correlationsdf = pd.concat(correlationsdf, axis=0)
-correlationsdf.to_parquet(plate_correlation_path / "well_agg_plate_genotype_correlations.parquet")
+correlationsdf.to_parquet(
+    plate_correlation_path / "well_agg_plate_genotype_correlations.parquet"
```

Comment:
should this parquet file depend on the `data_type` variable? (i.e., it should save a different file depending on what you set `data_type` as?)

Reply:
I was thinking about this and honestly I was just wanted to avoid adding more files but do you feel strongly about this one way or another? @gwaybio

Reply:
I can add it now honestly, won't take long :)

Reply:
i feel strongly that it should be clear what this file represents - is it qc data? I agree with less bloat, so if it's not super useful to have both, then only include one.